resume/resume.json

```diff
@@ -50,8 +50,8 @@
     },
     {
       "highlights": [
-        "Design and implementation of web applications for own needs organization",
-        "Design, implementation and maintenance of the entire environment IT organization",
+        "Design and implementation of web applications for internal needs of the organization",
```

Comment:
I have impresion that changes slighly changes meaning. Organisation have own needs pushed by external factors, because application is publicly available. Application which I have developed does not fullfill only internal process, but also externals stakeholder participates in it eg. fedrowanie.siecobywatelska.pl (governements sends us documents; we publish documents, so everybody can verify us), poradnia.siecobywatelska.pl (clients sends us requests for help, something like ticket system).